### PR TITLE
Athmospheric stabillity processor

### PR DESCRIPTION
**Pull request by ErlendHaa**

### CODE REVIEW

File: camille/processors/athmospheric_stabillity.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+
+def process(amb, sea):
+    """ TODO: option handling """
+
+    df = sea.resample('20T').mean() - amb.resample('20T').mean()
+    df['atm_stb'] = df.apply(lambda row: _stabillity_category(row[0]), axis=1)
```
Comment (JensGM) [inline camille/processors/athmospheric_stabillity.py]: You could apply directly to the column. Could do without the lambda then

File: tests/test_athmospheric_stabillity.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import unittest
+import pandas as pd
+import numpy as np
+
+from camille import athmospheric_stabillity
+
+
+amb_data = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
```
Comment (JensGM) [inline tests/test_athmospheric_stabillity.py]: We should get some real data

File: tests/test_athmospheric_stabillity.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            3, 3, 3, 3, 3, 3, 3 ,3 ,3 ,3 ,
+            4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+            6, 6, 6, 6, 6, 6 ,6, 6, 6, 6]
+
+
+class TestAthmosphericStabillity(unittest.TestCase):
```
Comment (JensGM) [inline tests/test_athmospheric_stabillity.py]: I think we should switch to use pytest

**Comment by ErlendHaa** [comment]
#17

File: camille/process/stb_cat.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import pandas as pd
```
Comment (JensGM) [inline camille/process/stb_cat.py]: I think this processor should be called stability_category

File: camille/process/stb_cat.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+def process(deltatemp):
+    stabillity_category = pd.Series(deltatemp.apply(_stabillity_category))
+    deltatemp.columns = ['AtmStb']
```
Comment (JensGM) [inline camille/process/stb_cat.py]: this seems incorrect

File: camille/process/delta_temp.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+def process(amb, sea):
+    delta_temp  = sea.resample('20T').mean() - amb.resample('20T').mean()
+    delta_temp.columns = ['delta_temp']
```
Comment (JensGM) [inline camille/process/delta_temp.py]: This shouldn't be needed? Is this a renaming?

Comment (ErlendHaa) [reply camille/process/delta_temp.py]: Do we not want a column name for the returned Series?

File: camille/process/stb_cat.py (hunk 5 above)
Comment (ErlendHaa) [reply camille/process/stb_cat.py]: Indeed!

**Pull request merged**